<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <node name="hsv_tuner" pkg="rr_iarrc" type="hsv_tuner" output="screen">
        <param name="values_topic" type="string" value= "/hsv_tuned"/>
    	<param name="load_file" type="string" value= "$(find rr_iarrc)/saved_hsv/example.txt"/>
    </node>
</launch>
